en/asgardeo/docs/guides/users/attributes/add-scim2-custom-user-schema-support.md

```diff
@@ -0,0 +1,422 @@
+4. Retrieve the user using SCIM and ensure that the custom claim value is present.
+
+    !!! abstract ""
+        **SCIM Request**
+        ```bash
+        curl -v -k --user admin:admin https://localhost:9443/scim2/Users
+        ```
+        ---
+        **SCIM Response**
+        ```
+        {
+            "totalResults": 1,
+            "startIndex": 1,
+            "itemsPerPage": 1,
+            "schemas": [
+                "urn:ietf:params:scim:api:messages:2.0:ListResponse"
+            ],
+            "Resources": [
+                {
+                    "emails": [
+                        "alex@wso2.com"
+                    ],
+                    "urn:scim:schemas:extension:custom:User": {
+                        "deviceNames": [
+                            "device1",
+                            "device2",
+                            "device3"
+                        ]
+                    },
+                    "meta": {
+                        "created": "2025-04-15T13:15:59.362739Z",
+                        "location": "https://localhost:9443/scim2/Users/65f09c38-23e0-4132-867f-451699e18022",
+                        "lastModified": "2025-04-15T18:01:55.805973Z",
+                        "resourceType": "User"
+                    },
+                    "name": {
+                        "givenName": "Alex",
+                        "familyName": "Parker"
+                    },
+                    "id": "65f09c38-23e0-4132-867f-451699e18022",
+                    "userName": "alex",
+                    "urn:scim:wso2:schema": {
+                        "emailAddresses": [
+                            "alex@wso2.com"
+                        ]
+                    }
+                }
+            ]
+        }
+        ```
+
+5. You can also try out updating the **Device Names** attribute values using the following SCIM command and check if the values are updated by accessing the user profile on the console. 
+
+    !!! abstract ""
+        **SCIM Patch Request**
+        ```
+        curl --location --request PATCH 'https://localhost:9443/scim2/Users/65f09c38-23e0-4132-867f-451699e18022' \
+        --header 'Authorization: Basic YWRtaW46YWRtaW4=' \
+        --header 'Content-Type: application/json' \
+        --data '{
+            "schemas": [
+                "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+            ],
+            "Operations": [
+                {
+                    "op": "replace",
+                    "value": {
+                        "urn:scim:schemas:extension:custom:User": {
+                            "deviceNames": [
+                                "new_device1",
+                                "new_device2"
+                            ]
+                        }
+                    }
+                }
+            ]
+        }'
+        ```
+
+
+## How to add claims to support Complex attributes
```

Comment:
Let's remove this complex attribute part from the asgardeo docs as the additional properties tab is not enabled in asgardeo.

Reply:
Fixed by - https://github.com/wso2/docs-is/pull/5256/commits/66a333b82e73b460f612eb1f51b0ba23bf81a75e